Mathlib/NumberTheory/IntervalSums.lean

```diff
@@ -0,0 +1,182 @@
+the same as the limit over open intervals `Ioo -N N` (see `SymmetricConditional_eq_map_Ioo`). -/
+abbrev SummationFilter.symCondInt : SummationFilter ℤ :=
+  SummationFilter.symmetricConditional ℤ
+
+lemma SymmetricConditional_eq_map_Icc :
+    symCondInt.filter = atTop.map (fun N ↦ Icc (-N) N) := rfl
+
+lemma SymmetricConditional_eq_map_Icc_nat :
+    symCondInt.filter = atTop.map (fun N : ℕ ↦ Icc (-(N : ℤ)) N) := by
+  rw [symCondInt, symmetricConditional, ← Nat.map_cast_int_atTop]
+  rfl
+
+/-- The SummationFilter on `G` corresponding to the intervals `Ico (-N) N`. -/
+def SummationFilter.IcoFilter : SummationFilter G where
+  filter := atTop.map (fun N ↦ Ico (-N) N)
+
+/-- The SummationFilter on `G` corresponding to the intervals `Ioc (-N) N`. -/
+def SummationFilter.IocFilter : SummationFilter G where
+  filter := atTop.map (fun N ↦ Ioc (-N) N)
+
+lemma SymmetricConditional_eq_map_Ioo :
+    (symmetricConditional ℤ).filter = atTop.map (fun N ↦ Ioo (-N) N) := by
+  rw [symmetricConditional, ← Nat.map_cast_int_atTop]
+  ext s
+  simp only [Filter.mem_map, mem_atTop_sets, ge_iff_le, Set.mem_preimage]
+  constructor
+  · intro ⟨a, ha⟩
+    refine ⟨a + 1, fun b hb ↦ ?_⟩
+    convert ha (b - 1) (by grind) using 1
+    ext x
+    rw [mem_Ioo, mem_Icc]
+    grind
+  · intro ⟨a, ha⟩
+    refine ⟨a - 1, fun b hb ↦ ?_⟩
+    convert ha (b + 1) (by grind) using 1
+    ext x
+    rw [mem_Icc, mem_Ioo]
+    grind
+
+variable [(atTop : Filter G).NeBot]
+
+instance : (symmetricConditional G).NeBot where
+  ne_bot := by simp [symmetricConditional, Filter.NeBot.map]
+
+instance : (IcoFilter G).NeBot where
+  ne_bot := by simp [IcoFilter, Filter.NeBot.map]
+
+instance : (IocFilter G).NeBot where
+  ne_bot := by simp [IocFilter, Filter.NeBot.map]
+
+variable {G : Type*} [AddCommGroup G] [PartialOrder G] [IsOrderedAddMonoid G] [LocallyFiniteOrder G]
+
+-- This proof was basically done by Aristotle
+lemma SymmetricConditional_le_Conditional :
+    (symmetricConditional G).filter ≤ (conditional G).filter := by
+  simp only [symmetricConditional, map_le_iff_le_comap, ← @tendsto_iff_comap, conditional]
+  suffices  Tendsto (fun g : G ↦ (-g, g)) atTop (atBot ×ˢ atTop) by
+    exact Filter.map_mono this
+  exact Filter.Tendsto.prodMk tendsto_neg_atTop_atBot (tendsto_id)
+
+instance : (symmetricConditional G).LeAtTop where
+  le_atTop := le_trans SymmetricConditional_le_Conditional (conditional G).le_atTop
+
+instance : (IcoFilter ℤ).LeAtTop where
+  le_atTop := by
+    rw [IcoFilter, map_le_iff_le_comap, ← @tendsto_iff_comap]
+    exact tendsto_Ico_atTop_atTop
+
+instance : (IocFilter ℤ).LeAtTop where
+  le_atTop := by
+    rw [IocFilter, map_le_iff_le_comap, ← @tendsto_iff_comap]
+    exact tendsto_Ioc_atTop_atTop
+
+variable {α : Type*} {f : ℤ → α} [CommGroup α] [TopologicalSpace α] [ContinuousMul α]
+
+@[to_additive]
+lemma multipliable_IcoFilter_of_multiplible_SymmetricConditional
```

Comment:
```suggestion
lemma multipliable_IcoFilter_of_multipliable_symmetricConditional
```
(if you hover over `to_additive` in the present version you'll see that the additivized lemma name is a weird mix, because it didn't recognise your typo as such!